File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.swift
import Foundation

func solution(_ n: Int) -> Int {
    var count = 0
    
    // 1부터 n까지 반복하여 a 값을 찾음
    for a in 1...n {
        // a가 n을 나누었을 때 나머지가 0이면 b는 자연수가 된다.
        if n % a == 0 {
            count += 1
        }
    }
    
    return count
}

File: 프로그래머스/0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {
    var answer = Array.from({ length: n }, (_, i) => i + 1)  // 1부터 n까지 배열 생성
                      .filter(a => n % a === 0)  // 약수만 필터링
                      .length;  // 약수의 개수 세기
    return answer;
}
